pkg/asset/installconfig/powervs/platform.go

```diff
@@ -0,0 +1,128 @@
+package powervs
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/core"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/installer/pkg/types/powervs"
+)
+
+// Platform collects powervs-specific configuration.
+func Platform() (*powervs.Platform, error) {
+	regions := knownRegions()
+	
+        // TODO(cklokman): This section came from aws and transforms the response from knownRegions
+        //                 into long and short regions to prompt the user for region select this section
+        //                 need need to be different based on powervs's implementation of knownRegions
+        //
+
+        longRegions := make([]string, 0, len(regions))
+	shortRegions := make([]string, 0, len(regions))
+	for id, location := range regions {
+		longRegions = append(longRegions, fmt.Sprintf("%s (%s)", id, location))
+		shortRegions = append(shortRegions, id)
+	}
+        
+	var regionTransform survey.Transformer = func(ans interface{}) interface{} {
+		switch v := ans.(type) {
+		case core.OptionAnswer:
+			return core.OptionAnswer{Value: strings.SplitN(v.Value, " ", 2)[0], Index: v.Index}
+		case string:
+			return strings.SplitN(v, " ", 2)[0]
+		}
+		return ""
+	}
+
+        // TODO(cklokman): We need to verify that us_south is the correct defaultRegion to use
+        //
+	defaultRegion := "us_south"
```

Comment:
i'm more leaning toward not having a default region at all

Reply:
I agree.  aws does allow for a default region, and I was just trying to maintain similar functionality.  This does however raise the question of why default regions are acceptable on aws and not on powervs.  I'm guessing it has something to do with live migration?  Or perhaps regions in which services or resources are available for powervs?  Just trying to nail down a reason to remove this feature.  I totally agree, spinning up a cluster in a region not specified... doesn't seem to be the direction we'd want to go.

Reply:
@clnperez any specific reason to avoid having a default region ?

Reply:
@23TNC this link is a good reference for region/zone  - https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server
The region naming is typically `us-south` , `us-east` etc

Reply:
@bpradipt no specific reason. it just seems like an easy enough thing to ask users for, and a possibility for a miserable experience if the default region happens to be far away. but if aws does it, then that's probably what the ocp team prefers so let's keep one

Reply:
the install-config does ask for regions but always provides a default where possible. this is just to make the user experience simple and having the user just go with a default.

Reply:
i was thinking of the case of the hand-built install-config.yaml. in that case, if you don't think about it, and leave it out, it picks one. in the case of the survey, it does show you which is the default, so at least you'll think, "oh no...that's a bad default for me." so that makes me feel better. most people will probably not hand-build their install configs.